README.md

```diff
@@ -1,2 +1,64 @@
-> Math-magician
-Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and also read random math-related quotes.
+# Math-magician
+> "Math magicians" It is just a simple project uses a Single Page App that enables users to make simple calculations and read a random math-related quote. 
```

Comment:
- [ ] Kindly deploy the project online and include the livelink in the readme.

Reply:
done